ga4gh/frontend.py

```diff
@@ -93,6 +93,10 @@ def handleHttpOptions():
 
 @app.errorhandler(Exception)
 def handleException(exception):
```

Comment:
This method is quite tricky now. I think it's find, but I'd have to search about quite hard to figure out why things are the way they are. Can we get some comments to explain the reason for each of the separate chunks?